rllib/connectors/common/add_states_from_episodes_to_batch.py

```diff
@@ -367,12 +389,17 @@ def __call__(
                     continue
 
                 # Episode just started -> Get initial state from our RLModule.
-                if sa_episode.t_started == 0 and len(sa_episode) == 0:
+                if (
+                    sa_episode.t_started == 0
+                    and len(sa_episode) == 0
+                    or (Columns.STATE_OUT not in sa_episode.extra_model_outputs)
```

Comment:
Same here, when would this be the case (STATE_OUT missing)?

Let's comment.

Reply:
I think the condition was also ill-suited. It needed some further pair of brackets.